From e47b47f64e0f9188c4247700825f67d044e6ea56 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Fri, 8 Aug 2014 00:19:45 +0200
Subject: [PATCH] TWRP: Add display flags

---
 ramdisk/twrp.fstab | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ramdisk/twrp.fstab b/ramdisk/twrp.fstab
index bbc038b..eb190d2 100644
--- a/ramdisk/twrp.fstab
+++ b/ramdisk/twrp.fstab
@@ -6,7 +6,7 @@
 /system                      ext4                 /dev/block/mmcblk0p19                                  flags=display="System";backup=1;wipeingui
 /data                        ext4                 /dev/block/mmcblk0p21                                  length=-16384 flags=display="Data";backup=1;wipeingui;wipeduringfactoryreset
 /preload                     ext4                 /dev/block/mmcblk0p20                                  flags=backup=1
-/sdcard                      datamedia            /dev/null                                              flags=storage
-/external_sd                 auto                 /dev/block/mmcblk1p1       /dev/block/mmcblk1          flags=removable;storage
+/sdcard                      datamedia            /dev/null                                              flags=display="Internal Storage";storage
+/external_sd                 auto                 /dev/block/mmcblk1p1       /dev/block/mmcblk1          flags=display="External Storage";removable;storage
 /usb-otg                     auto                 /dev/block/sda1            /dev/block/sda              flags=display="USB-OTG";storage;wipeingui;removable
 /modem                       emmc                 /dev/block/mmcblk0p7
-- 
2.0.1

